app/main.py

```diff
@@ -1 +1,15 @@
 # write your code here
+def copy_file(command: str) -> None:
+    try:
+        command_cp, file_name, file_copy_name = command.split()
```

Comment:
Please rename variables.
command_cp -> In this step we can not say that it is cp_command. Just command or input_command
file_name -> Why need file_name? input_file_name or source_file_name
file_copy_name -> Why need file_copy_name? output_file_name or destination_file_name